src/Schema/Comparator.php

```diff
@@ -20,31 +21,29 @@
  */
 class Comparator
 {
-    /**
-     * @return SchemaDiff
-     *
-     * @throws SchemaException
-     */
-    public static function compareSchemas(Schema $fromSchema, Schema $toSchema)
```

Comment:
Would it be possible to retain the original placement of the methods? Otherwise, it takes extra effort to confirm that the method hasn't been actually removed but was put in a different place in the class.

Reply:
The methods are in the same order before and after the PR: `compareSchemas()` first, then `compare()`.

It looks like Git shows the diff this way because the long body moved from `compare()` to `compareSchemas()`.